TUTORIAL_FROMXML.md

```diff
@@ -13,7 +13,7 @@ To invoke the generator we will use the following test:
     @Test
     public void testBeanGenerator() throws Exception {
         if (new Geci()
-            .source("./src/test/resources", "./javageci-examples/src/test/resources")
+            .source(maven().testResources(), maven().module("javageci-examples").testResources())
```

Comment:
Please check if that works really. I think it does not.

`maven().module("").xxx()` returns something that is already a list of directories. Read the documentation on how the sources are scanned and how the directories are searched.

Reply:
Reverted.